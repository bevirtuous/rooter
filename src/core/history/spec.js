import { createBrowserHistory } from 'history';
import history from '.';

describe('history', () => {
  it('should be a `createBrowserHistory` instance', () => {
    expect(history).toBe(createBrowserHistory);
  });
});
