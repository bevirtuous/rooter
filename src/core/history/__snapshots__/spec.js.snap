// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`history should be a \`createBrowserHistory\` instance 1`] = `
Object {
  "action": "POP",
  "block": [Function],
  "createHref": [Function],
  "go": [Function],
  "goBack": [Function],
  "goForward": [Function],
  "length": 1,
  "listen": [Function],
  "location": Object {
    "hash": "",
    "pathname": "/myroute/123",
    "search": "",
    "state": undefined,
  },
  "push": [Function],
  "replace": [Function],
}
`;
